scrapy/core/scheduler.py

```diff
@@ -107,7 +108,23 @@ def _dqpush(self, request):
             return
         try:
             self.dqs.push(request)
-        except ValueError as e:  # non serializable request
+        except TransientError as e:
+            msg = "Unable to push request to queue: %s"
+            logger.warning(msg, e, exc_info=True, extra={'spider': self.spider})
```

Comment:
What do you think about adding a stats counter for this, similar to `scheduler/unserializable`?